WordPress/Classes/ViewRelated/Post/PrepublishingViewController.swift

```diff
@@ -31,10 +31,42 @@ class PrepublishingViewController: UITableViewController {
 
     private let completion: (AbstractPost) -> ()
 
+    private func rowTitle(rowID: PrepublishingIdentifier) ->  NSAttributedString {
+        switch rowID {
+        case .visibility:
+            return NSAttributedString(string: NSLocalizedString("Visibility", comment: "Label for Visibility"), attributes: nil)
+        case .schedule:
+            return NSAttributedString(string: Constants.publishDateLabel, attributes: nil)
+        case .tags:
+            let font = UIFont.systemFont(ofSize: 12.0)
+            let attributes: [NSAttributedString.Key: Any] = [
+                .font: font,
+                .foregroundColor: UIColor.textSubtle,
+            ]
+
+            let firstString = NSMutableAttributedString(string: "Tags ", attributes: nil)
+            let secondString = NSAttributedString(string: "(Grow your audience)", attributes: attributes)
```

Comment:
Some other possibilities besides _**Grow your audience**_ :

**_Make your content discoverable_**
**_Allow others to discover your content_**

I like "discoverable" because it ties in to the Discover tab. 

https://wordpress.com/support/posts/tags/
> Your posts will appear in the topic listings of any tags or categories you use. Therefore, assigning tags and categories to your post increases the chance that other WordPress.com users will see your content.

**_Increase engagement_**

Short and simple. I wonder if "engagement" is too jargon-y though.